src/app/modules/backbone/components/samples-confirmation-modal/samples-confirmation-modal.component.html

```diff
@@ -0,0 +1,17 @@
+<app-modal [display]="display">
```

Comment:
This is a bit weird since we're binding `ModalComponent`'s `display` attribute to itself. The reason I did this is that the modal fades out when it's closed, but if the css class which controls the opacity is bound to the `display` attribute of its respective component, this will be the `display` attribute of the component that Angular creates when it sees `<app-modal>` in the HTML of `SamplesConfirmationModalComponent` (`SCMComponent`), which is a different `display` attribute from the one in `SCMComponent` even though `SCMComponent` is a subclass of `ModalComponent`. Which I admit is confusing. I've tried to explain a little better in the diagram below.

Without binding `display`:
```
<div [class.open]="display" ...> ... <!-- In shared/.../modal.component.html-->
                       \
ModalComponent          \  ModalComponent (created by Angular when it sees <app-modal> in modules/.../samples-confirmation-modal.component.html)
┌─────────┐              \  ┌─────────┐
│ display │               ->│ display │
└─────────┘                 └─────────┘
     ^
     | extends
     |
SCMComponent (created by us when the user clicks on the floating action button)
┌─────────┐
│...      │
└─────────┘
````

However, when we do this binding, we ask `ModalComponent` to look at the `display` attribute of the `SCMComponent` we created, not its own one.